{"trial_id": "0", "hyperparameters": {"space": [{"class_name": "Choice", "config": {"name": "units1", "default": 8, "conditions": [], "values": [8, 16, 32, 64, 128], "ordered": true}}, {"class_name": "Choice", "config": {"name": "units2", "default": 8, "conditions": [], "values": [8, 16, 32, 64, 128], "ordered": true}}, {"class_name": "Choice", "config": {"name": "units3", "default": 8, "conditions": [], "values": [8, 16, 32, 64, 128], "ordered": true}}], "values": {"units1": 32, "units2": 8, "units3": 16}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"C:\\Users\\jack_\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\keras_tuner\\engine\\base_tuner.py\", line 266, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"C:\\Users\\jack_\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\keras_tuner\\engine\\base_tuner.py\", line 231, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\jack_\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\keras_tuner\\engine\\tuner.py\", line 282, in run_trial\n    self._configure_tensorboard_dir(callbacks, trial, execution)\n  File \"C:\\Users\\jack_\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\keras_tuner\\engine\\tuner.py\", line 381, in _configure_tensorboard_dir\n    for callback in callbacks:\nTypeError: 'EarlyStopping' object is not iterable\n"}